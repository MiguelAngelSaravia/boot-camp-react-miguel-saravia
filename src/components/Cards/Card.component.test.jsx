import React from 'react';
describe('custom value', () => {
    it('sums 2 values', () => {
        const num1 = 10
        const num2 = 10

        const result  = num1 + num2;
        expect(result).toBe(20);
    })
});
