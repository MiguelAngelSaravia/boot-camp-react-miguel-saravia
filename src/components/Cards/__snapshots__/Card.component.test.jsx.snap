// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomCard match snapshot CustomCard 1`] = `ShallowWrapper {}`;
